rln-wasm/tests/rln-wasm.rs

```diff
@@ -0,0 +1,272 @@
+#![cfg(target_arch = "wasm32")]
+
+#[cfg(test)]
+mod tests {
+    use js_sys::{BigInt as JsBigInt, Date, Object, Uint8Array};
+    use rln::circuit::{Fr, TEST_TREE_HEIGHT};
+    use rln::hashers::{hash_to_field, poseidon_hash};
+    use rln::poseidon_tree::PoseidonTree;
+    use rln::protocol::{prepare_verify_input, rln_witness_from_values, serialize_witness};
+    use rln::utils::{bytes_le_to_fr, fr_to_bytes_le};
+    use rln_wasm::*;
+    use wasm_bindgen::{prelude::*, JsValue};
+    use wasm_bindgen_test::wasm_bindgen_test;
+    use zerokit_utils::merkle_tree::merkle_tree::ZerokitMerkleTree;
+
+    #[wasm_bindgen(module = "src/utils.js")]
+    extern "C" {
+        #[wasm_bindgen(catch)]
+        fn read_file(path: &str) -> Result<Uint8Array, JsValue>;
+
+        #[wasm_bindgen(catch)]
+        async fn calculateWitness(circom_path: &str, input: Object) -> Result<JsValue, JsValue>;
+    }
+
+    #[cfg(feature = "arkzkey")]
+    const ZKEY_PATH: &str = "../rln/resources/tree_height_20/rln_final.arkzkey";
+    #[cfg(not(feature = "arkzkey"))]
+    const ZKEY_PATH: &str = "../rln/resources/tree_height_20/rln_final.zkey";
+
+    const CIRCOM_PATH: &str = "../rln/resources/tree_height_20/rln.wasm";
+
+    #[wasm_bindgen_test]
+    pub async fn rln_wasm_benchmark() {
+        let mut results = String::from("\nbenchmarks:\n");
+        let iterations = 10;
+
+        let zkey = read_file(&ZKEY_PATH).unwrap();
```

Comment:
could you please go trough the rln-wasm test and replace unwrap() to expect(). Especially on RLN related function